jvm-packages/README.md

```diff
@@ -61,6 +61,8 @@ if you want to use `xgboost4j-spark`, you just need to replace xgboost4j with `x
 
 ## Examples
 
+latest_version_num=0.72
```

Comment:
would you mind also modifying the doc by adding the instructions to refer the dependency? https://github.com/dmlc/xgboost/blob/master/doc/jvm/index.md

Reply:
Sure. I will do.

Reply:
in all places containing hard-coded version number, we might want to add note like "please check [Here](https://github.com/dmlc/xgboost/releases) for the latest release number"